scripts/docs/generate_asset_documentation.py

```diff
@@ -420,12 +418,12 @@ def save(self):
         if self._is_root:
             doc.add_heading(self._type.capitalize() + "s", level=1)
         else:
-            doc.add_heading(self._name.capitalize(), level=1)
+            doc.add_heading(self._name, level=1)
 
         if self._sub_categories:
-            self._sub_categories = dict(sorted(self._sub_categories.items(), key=lambda i: i[0].lower()))
+            sorted_sub_categories = dict(sorted(self._sub_categories.items(), key=lambda i: i[0].lower()))
```

Comment:
You'll probably need an OrderedDict instead of a regular one, to maintain the order in which the items were inserted: https://docs.python.org/3/library/collections.html#collections.OrderedDict

Reply:
Would it be ordered if we're using Python 3.8? https://stackoverflow.com/questions/34305003/difference-between-dictionary-and-ordereddict

Reply:
Nice find! Looks like we're all set, then!